src/main/java/org/apache/jenkins/gitpubsub/GitPubSubPoll.java

```diff
@@ -223,7 +223,7 @@ public void onThrowable(Throwable t) {
         public synchronized STATE onBodyPartReceived(HttpResponseBodyPart content) throws Exception {
             lastTime = System.currentTimeMillis();
             byte[] body = content.getBodyPartBytes();
-            if (body.length < 2 || body[body.length - 1] != 0x0a || body[body.length - 2] != 0x0d) {
+            if (body.length < 2 || body[body.length - 1] != 0x0a) {
```

Comment:
Technically we could change to 
```suggestion
            if (body.length < 1 || body[body.length - 1] != 0x0a) {
```
But unclear that a 1 character message is meaningful javascript, so not required